api/spec/services/pdfs/expense_sheet/generator_service_spec.rb

```diff
@@ -86,18 +89,33 @@
 
       context 'when driving_expenses_comment is empty' do
         let(:expense_sheet_data) do
-          {
-            beginning: Date.parse('2018-01-01'),
-            ending: Date.parse('2018-01-27'),
+          expense_sheet_data_defaults.merge(
             driving_expenses_comment: nil,
             user: service.user
-          }
+          )
         end
 
         it 'renders correct text' do
           expect(pdf_text_inspector.strings[-12]).to eq 'Keine Angaben'
         end
       end
+
+      context 'when extraordinary_expenses is not empty' do
+        let(:expense_sheet_data) do
+          expense_sheet_data_defaults.merge(
+            extraordinary_expenses: -15_000,
```

Comment:
Bit wired test case as is negative but yes, i know they use it for minus expenses so I guess is ok